### DISCUSSION TITLE
I run the dpgen ，and here are the errors

### DISCUSSION THREAD

**Discussion by wzdns** [deepmodeling/dpgen#1536] [Q&A]
I run the dpgen ，and here are the errors .What should I do
![1714903147485](https://github.com/deepmodeling/dpgen/assets/168963956/7b607380-2bff-4edd-bb2c-4fc0491deaa8)

**Comment by njzjz** [comment]
As indicated in the error message, your [JSON](https://www.json.org/json-en.html) is not valid.
